File: app/src/main/java/com/android/learn/mcontract/MainActivityContract.java
package com.android.learn.mcontract;

import com.android.base.mmodel.HotKeyData;

import java.util.List;

/**
 * Created by gaolei on 2018/6/18.
 */

public class MainActivityContract {

    public interface Presenter {

        void getHotKey();

    }

    public interface View {

        void showHotKey(List<HotKeyData> list);

    }
}

File: app/src/main/java/com/android/learn/mcontract/MainActivityContract.kt
package com.android.learn.mcontract

import com.android.base.mmodel.HotKeyData
import com.android.base.mview.BaseView

/**
 * Created by gaolei on 2018/6/18.
 */

class MainActivityContract {

    interface Presenter {

        fun getHotKey()

    }

    interface View : BaseView {

        fun showHotKey(list: List<HotKeyData>)

    }
}
